packages/plugin-npm/sources/configurationUtils.ts

```diff
@@ -0,0 +1,42 @@
+import {Configuration, Ident} from '@berry/core';
+
+function getScopedConfiguration(ident: Ident, configuration: Configuration): Configuration|Map<string, any> {
+  if (ident.scope) {
+    const scopeConfigurations: Map<string, Map<string, any>>|null = configuration.get(`npmScopes`);
+
+    if (scopeConfigurations && scopeConfigurations.has(ident.scope)) {
```

Comment:
Do we expect scoped configuration to have the `@` prefix or not? i.e. do we want

```yaml
npmScopes:
   private-scope:
     # ...
```

or

```yaml
npmScopes:
   "@private-scope":
     # ...
```

?

I've implemented both, but I'd prefer to have only one. Not sure which though, I like the `@` version because scopes start with `@`, but the downside of adding the `@` is that we must put quotes around it with the current syml parser.

Reply:
The `parseIdent` function generates them without the leading`@` so I guess we should keep this convention.